Source/Krypton Components/Krypton.Toolkit/Rendering/ThemeManager.cs

```diff
@@ -121,14 +123,14 @@ public class ThemeManager
         /// </summary>
         /// <param name="themeName">Name of the theme.</param>
         /// <param name="manager">The manager.</param>
-        public static void ApplyTheme(string themeName, KryptonManager manager) => ApplyTheme(_supportedThemes.GetByFirst(themeName), manager);
+        public static void ApplyTheme(string? themeName, KryptonManager manager) => ApplyTheme(_supportedThemes.GetByFirst(themeName), manager);
 
         /// <summary>
         /// Sets the theme.
         /// </summary>
         /// <param name="themeName">Name of the theme.</param>
         /// <param name="manager">The manager.</param>
-        public static void SetTheme(string themeName, KryptonManager manager)
+        public static void SetTheme(string? themeName, KryptonManager manager)
```

Comment:
Needs to be `non` nullable to allow the code to be cleaner.

Reply:
Undo changes made by ReSharper